source/Mlos.Python/mlos/Optimizers/ExperimentDesigner/NumericOptimizers/RandomSearchOptimizer.py

```diff
@@ -64,10 +64,19 @@ def suggest(self, context_values_dataframe=None):  # pylint: disable=unused-argu
         :return:
         """
 
-        feature_values_dataframe = self.optimization_problem.parameter_space.random_dataframe(num_samples=self.config.num_samples_per_iteration)

```

Comment:
We'll need to repeat this logic for glowworm and soon for random near incumbent. I wonder if we could keep it in the UtilityFunctionOptimizer base class to avoid duplication.

Reply:
this is not on github yet, but will do as soon as it is.